content/recipes/terminus.md

```diff
@@ -1,114 +1,137 @@
   }
 }
 @@switch
-import { Injectable, Dependencies } from '@nestjs/common';
-import { DNSHealthIndicator } from '@nestjs/terminus';
-
-@Injectable()
-@Dependencies(DNSHealthIndicator)
-export class TerminusOptionsService {
-  constructor(dns) {
-    this.dns = dns;
-  }
-
-  createTerminusOptions() {
-    const healthEndpoint = {
-      url: '/health',
-      healthIndicators: [
-        async () => this.dns.pingCheck('google', 'https://google.com'),
-      ],
-    };
-    return {
-      endpoints: [healthEndpoint],
-    };
+@Controller('health')
+@Dependencies(HealthCheckService, DNSHealthIndicator)
+export class HealthController {
+  constructor(
+    private health,
+    private dns,
+  ) { }
+
+  @Get()
+  @HealthCheck()
+  healthCheck() {
+    return this.health.check([
+      async () => this.dns.pingCheck('nestjs-docs', 'https://docs.nestjs.com'),
+    ])
   }
 }
 ```
 
-Once we have set up our `TerminusOptionsService`, we can import the `TerminusModule` into the root `ApplicationModule`. The `TerminusOptionsService` will provide the settings, which in turn will be used by the `TerminusModule`.
+Our health check will now send a *GET*-request to the `https://docs.nestjs.com` address. In case
+we get a healthy response from that address, our route at `http://localhost:3000/health` will return
+the following object with the 200 status code.
 
-```typescript
-@@filename(app.module)
-import { Module } from '@nestjs/common';
-import { TerminusModule } from '@nestjs/terminus';
-import { TerminusOptionsService } from './terminus-options.service';
-
-@Module({
-  imports: [
-    TerminusModule.forRootAsync({
-      useClass: TerminusOptionsService,
-    }),
-  ],
-})
-export class ApplicationModule { }
+```json
+{
+  "status": "ok",
+  "info": {
+    "nestjs-docs": {
+      "status": "up"
+    }
+  },
+  "error": {},
+  "details": {
+    "nestjs-docs": {
+      "status": "up"
+    }
+  }
+}
 ```
 
-> info **Hint** If done correctly, Nest will expose the defined health check(s), which are reachable through a GET request to the defined route. For example `curl -X GET 'http://localhost:3000/health'`
+The interface of this response object can be accessed from the `@nestjs/terminus` package with the `HealthCheckResult`-interface.
+
+|           |                                                                                                                                                                                             |                                      |
+|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------|
+| `status`  | In case any health indicator failed the status will be `error`. If the NestJS app is shutting down but still accepting HTTP requests, the health check will have the `shutting_down` status. | `'error' \| 'ok' \| 'shutting_down'` |
```

Comment:
`error` -> `'error'` 
and
`shutting_down` -> `'shutting_down'`

also, same for `up` and `down` below

Reply:
Resolved, thanks <3